quinn-proto/src/crypto.rs

```diff
@@ -147,6 +140,18 @@ pub trait CryptoClientConfig {
     ) -> Result<Self::Session, ConnectError>;
 }
 
+impl CryptoServerConfig for Arc<ServerConfig> {
```

Comment:
Why for `Arc<ServerConfig>` instead of for `ServerConfig`?

Reply:
See `ClientConfig`.